packages/dappkit/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@celo/dappkit",
+  "version": "0.0.1-j",
+  "scripts": {
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@celo/contractkit": "0.0.3",
+    "@celo/utils": "0.0.1-dappkit6",
```

Comment:
What's with the subversioning here?

Reply:
I had to try a bunch of package versions but did not want to "officially" bump the version